sqlglot/dialects/spark.py

```diff
@@ -41,6 +41,17 @@ class Parser(Spark2.Parser):
         }
 
     class Generator(Spark2.Generator):
+        TYPE_MAPPING = {
+            **Spark2.Generator.TYPE_MAPPING,
+            exp.DataType.Type.TINYINT: "TINYINT",
+            exp.DataType.Type.SMALLINT: "SMALLINT",
+            exp.DataType.Type.BIGINT: "BIGINT",
+            exp.DataType.Type.MONEY: "DECIMAL(15, 4)",
+            exp.DataType.Type.SMALLMONEY: "DECIMAL(6, 4)",
+            exp.DataType.Type.VARBINARY: "BINARY",
+            exp.DataType.Type.BIT: "BOOLEAN",
```

Comment:
is this and uniqueidentifier correct?

Reply:
The `BIT` entry is redundant for the same reason as `VARBINARY`. The `UNIQUEIDENTIFIER` entry doesn't seem correct per T-SQL's docs: https://learn.microsoft.com/en-us/sql/t-sql/data-types/uniqueidentifier-transact-sql?view=sql-server-ver16?

Reply:
Double checking on our recommendations for `UNIQUEIDENTIFIER`

Reply:
@GeorgeSittas I see now you're saying VARBINARY and BIT are redundant with the HIVE definition from which SPARK inherits. They are removed now.